src/app/context/developer_mode/DeveloperModeProvider.tsx

```diff
@@ -14,9 +13,9 @@ import { useMutation, useQuery } from '@tanstack/react-query';
 import { ReactNode } from 'react';
 import { PrimaryButton } from '../../components/button.js';
 import { InfoDialog } from '../../modals/InfoDialog.js';
-import { TOKEN_LIST } from '../../model/token_list.js';
-import { DeveloperContext } from './DeveloperContext.js';
+import { TOKEN_LIST, deployTestTokens } from '../../model/token_list.js';
```

Comment:
Just a random thought: How about using [Typescript paths](https://www.typescriptlang.org/tsconfig#paths) so that we can have
```
import { TOKEN_LIST, deployTestTokens } from '@model/token_list.js';
```
instead of relative paths.